src/app/lib/utilities/getRemoteConfig/index.test.js

```diff
@@ -0,0 +1,104 @@
+import fetchMock from 'fetch-mock';
+import { CONFIG_REQUEST_RECEIVED, CONFIG_FETCH_ERROR } from '#lib/logger.const';
+
+const mockUrl =
+  'https://mock-config-endpoint?application=simorgh&service=mundo&__amp_source_origin=http://localhost';
+const mockResponse = { testToggle: { enabled: true } };
+fetchMock.mock(mockUrl, mockResponse);
+
+describe('getRemoteConfig', () => {
+  beforeEach(async () => {
+    jest.resetModules();
+    await import('#testHelpers/loggerMock');
+    process.env.SIMORGH_CONFIG_URL = 'https://mock-config-endpoint';
+  });
+
+  it('should return null if enableFetchingToggles is not enabled', async () => {
+    jest.mock('#lib/config/toggles', () => ({
+      local: {
+        enableFetchingToggles: { enabled: false },
+      },
+    }));
+
+    // Dynamic import is used in these tests so the toggles file values can be changed
+    const getRemoteConfig = await import('.');
```

Comment:
To change the values of https://github.com/bbc/simorgh/blob/0dbdccef9c05da94ecd68d590fb75fc092028a58/src/app/lib/config/toggles/index.js the best solution seemed to be dynamic import. It might be better if `#lib/config/toggles` returned a function so it's easier to mock, but wasn't sure I should include such refactors.